src/introduction.apib

```diff
@@ -1,6 +1,4 @@
-This is the new Teamleader API. We will make additions and changes to this API, but don't worry; we will keep the documented endpoints backwards compatible for the lifetime of this major API version.
-
-Do you have any feedback, do you miss functionality to support your use case? Let us know via [api@teamleader.eu](mailto:api@teamleader.eu).
+This is the new Teamleader API.  If you have any feedback, or are you missing functionality to support your use case? Let us know via [api@teamleader.eu](mailto:api@teamleader.eu).
```

Comment:
Should we still call it "new"?

Reply:
"This is the documentation of the Teamleader API." ?